### DISCUSSION TITLE
Why is browserlist OR instead of AND?

### DISCUSSION THREAD

**Discussion by dominictobias** [browserslist/browserslist#797] [Q&A] [answered]
I was suprised that this:

```
last 2 years
> 0.25%
not dead
not ie 11
not op_mini all
```

Is outputting a much larger babel bundle than this:
```
last 2 years
```

It makes no sense to me that these statements are not AND?

**Comment by ai** [comment, marked as answer]
Yes, they are `OR` for `last 2 years` and `> 0.25%` and `AND NOT` for `not …` queries.

They are not `AND` because in that case this config will produce 0 browsers:

```
chrome 11
chrome 10
```

Of course in that the most popular use case with `AND` it will be `chrome 11 AND chrome 10 = []`.

You need query [`last 2 versions and > 0.25% …`](https://browsersl.ist/#q=last+2+years+and+%3E+0.25%25%0Anot+dead%0Anot+ie+11%0Anot+op_mini+all)

**Reply by dominictobias** [reply]
Aha thanks!
